project-base/storefront/components/Layout/CustomerLayout.tsx

```diff
@@ -0,0 +1,47 @@
+import { UserNavigation } from 'components/Blocks/UserNavigation/UserNavigation';
+import { Breadcrumbs } from 'components/Layout/Breadcrumbs/Breadcrumbs';
+import { CommonLayout, CommonLayoutProps } from 'components/Layout/CommonLayout';
+import { Webline } from 'components/Layout/Webline/Webline';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import { TypeBreadcrumbFragment } from 'graphql/requests/breadcrumbs/fragments/BreadcrumbFragment.generated';
+import useTranslation from 'next-translate/useTranslation';
+import { useIsUserLoggedIn } from 'utils/auth/useIsUserLoggedIn';
+import { getInternationalizedStaticUrls } from 'utils/staticUrls/getInternationalizedStaticUrls';
+import { twMergeCustom } from 'utils/twMerge';
+
+type CustomerLayoutProps = {
+    pageTitle?: string;
```

Comment:
I think `title` is part of `CommonLayoutProps`, right? Maybe you could use that?

Reply:
Renamed for better understanding, it's a page heading, not a title like in CommonLayout